<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="6,2">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>4</number>
    </property>
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>4</number>
    </property>
    <property name="bottomMargin">
     <number>4</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="6,2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="labelImage">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(100, 100, 100);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_start"/>
   <addaction name="action_stop"/>
   <addaction name="action_reset"/>
  </widget>
  <action name="action_start">
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="action_stop">
   <property name="text">
    <string>stop</string>
   </property>
  </action>
  <action name="action_reset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
